### PR TITLE
workflows: Fix the fix to concurrency groups

### PR DESCRIPTION
**Pull request by nbusseneau**
This is a follow-up PR to #18193 :)

### CODE REVIEW

File: .github/workflows/conformance-aws-cni.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ concurrency:
     ${{
       (github.event_name == 'schedule' && github.sha) ||
       (github.event_name == 'issue_comment' && (
-        github.event.comment.body == '/ci-awscni') ||
+        github.event.comment.body == '/ci-awscni' ||
```
Comment (pchaigno) [inline .github/workflows/conformance-aws-cni.yaml:59]: And to think I did a review specifically looking at unmatched parenthesis :facepalm:

File: .github/workflows/conformance-aks-v1.10.yaml [hunk 2]
```diff
@@ -48,9 +48,7 @@ concurrency:
   # Note: for `issue_comment` triggers, we additionally need to filter out based
   # on comment content, otherwise any comment will interrupt workflow runs.
   group: |
-    ${{ github.workflow }}
-    ${{ github.event_name }}
-    ${{
+    ${{ github.workflow }} ${{ github.event_name }} ${{
```
Comment (pchaigno) [inline .github/workflows/conformance-aks-v1.10.yaml]: Where they not already below 400 characters for GitHub? I checked the GitHub Actions panel in my fork and it wasn't reporting errors anymore :-/

**Review by pchaigno — APPROVED** [review]
Thanks!

File: .github/workflows/conformance-aks-v1.10.yaml (hunk 2 above)
Comment (nbusseneau) [reply .github/workflows/conformance-aks-v1.10.yaml]: I think I might have been misled by the failing actions from PRs that were not rebased on `master`. I realized this while writing the commit description, so I put the PR on draft and am currently testing on my fork. If I indeed got misled I will drop the commit 😅

Comment (nbusseneau) [reply .github/workflows/conformance-aks-v1.10.yaml]: Indeed, confirmed it works and is only due to unrebased PRs. Sorry for the confusion, removing second commit :)

**Review by pchaigno — APPROVED** [review]

**Pull request merged**